emotion.md

```diff
@@ -0,0 +1,81 @@
+# Using Emotion
+
+[Emotion](https://emotion.sh/) is a CSS-in-JS library that provides component-scoping, encapsulation and composition for your styles.
+
+## Getting started
+
+```shell
+$ yarn add @emotion/react
+```
+
+> You need to install `@emotion/react` to use [Source components](https://github.com/guardian/source)
+
+## Configuration
+
+### TypeScript
+
+We recommend using [React's automatic runtime](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).
+
+In your `tsconfig.json`:
+
+```json
+{
+  "compilerOptions": {
+    "jsx": "react-jsx",
+    "jsxImportSource": "@emotion/react"
+  }
+}
+```
+
+> This tells TypeScript to use React's automatic runtime and lets TypeScript know where it can find the JSX namespace.
+
+### Babel
+
+You can use the `css` prop, in conjunction with `@emotion/babel-preset-css-prop`:
```

Comment:
do you need this if you have the above ts setup?

Reply:
No, I believe this is handled by the [`jsxImportSource` option](https://emotion.sh/docs/typescript#emotionreact) in your tsconfig's `compilerOptions`:

```
{
  "jsxImportSource": "@emotion/react"
}
```

Reply:
Should we make it clearer that TypeScript and Babel are 2 different paths for configuration? e.g

```
## Configuration

Emotion may be configured in your `tsconfig.json` or your `.babelrc`, depending on your build setup.

### Approach 1: TypeScript

...

### Approach 2: Babel

...
```

Reply:
👍

Reply:
added a suggestion so will resolve this